pages/api/prescriptions/update.ts

```diff
@@ -0,0 +1,43 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      const { id, name, status, pickupTime, balance, locationId, lockerBoxId } =
+        req.body
+
+      let prescription = null
+      prescription = await prisma.prescription.update({
```

Comment:
since we're not reassigning prescription it can just be `const prescription = await prisma...`